app/models/patron.rb

```diff
@@ -12,6 +20,27 @@ def key
     record['key']
   end
 
+  def first_name
+    fields['firstName']
+  end
+
+  def last_name
+    fields['lastName']
+  end
+
+  def status
+    if expired?
+      'Expired'
+    else
+      PATRON_STANDING.fetch(fields['standing']['key'], '')
+    end
+  end
+
+  def expired?
+    expired_date = fields['privilegeExpiresDate']
+    expired_date.present? ? Time.zone.now.strftime('%Y-%m-%d') >= expired_date : false
```

Comment:
If nothing else, it'd let us do a direct comparison here, instead of a string compare, maybe something like:

```suggestion
    expired_date && Time.zone.now >= expired_date
```